app/classes/document_query.rb

```diff
@@ -91,12 +91,28 @@ def boosted_fields
 
   def functions
     [
-      { gauss: { created: { origin: 'now', scale: '1825d', offset: '30d', decay: 0.3 } } },
-      { filter: {
+      # Prefer more recent documents
+      {
```

Comment:
Note: I adjusted the bracket positioning for these functions to be more consistent with their use in ES JSON.